### PR TITLE
Infix Convertor & Tree Sort

### PR DESCRIPTION
**Pull request by adiXcodr**
This PR will add these algorithms (C++) -

1. Infix to Prefix Converter.
2. Infix to Postfix Converter.
3. Binary Search Tree Sort

### CODE REVIEW

File: Algorithms/Sorting/tree_sort.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#include<bits/stdc++.h>
```
Comment (Bhavnaharitsa) [inline Algorithms/Sorting/tree_sort.cpp:1]: add algorithm name, description sample input output for each algorithm

File: Algorithms/Sorting/tree_sort.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+		  insert(root, arr[i]); 
+
+	int i = 0; 
+	storeSorted(root, arr, i); 
+} 
+
+
+int main() 
+{ 
+  int n;
```
Comment (Bhavnaharitsa) [inline Algorithms/Sorting/tree_sort.cpp:61]: what is n?

File: Algorithms/Stack/infix_to_prefix.cpp [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      // Final prefix expression is 
+      // present in operands stack. 
+      return operands.top(); 
+  } 
+
+
+    int main()
+    {
+      int value;
+      string s;
```
Comment (Bhavnaharitsa) [inline Algorithms/Stack/infix_to_prefix.cpp:143]: no single letter variable

**Comment by Bhavnaharitsa** [comment]
In all the algorithms, add algorithm name, description, sample input output and the changes I have mentioned and we are good to go

**Comment by adiXcodr** [comment]
> In all the algorithms, add algorithm name, description, sample input output and the changes I have mentioned and we are good to go

Sure,

**Pull request closed without merging**